src/Accordion/index.js

```diff
@@ -64,19 +64,29 @@ export default withStyles((theme) => ({
     display: "flex",
     flexDirection: "column",
     borderBottom: `1px solid ${theme.colors["gray200"]}`,
+    transition: "all 2s ease-in-out",
```

Comment:
Might be worthwhile to add this transition to the transitions theme object. `theme.transition.accordion`